xla/service/gpu/gpu_hlo_schedule.cc

```diff
@@ -624,7 +626,20 @@ Status ScheduleGpuModule(HloModule* module, int64_t pointer_size,
   std::unique_ptr<LatencyEstimator> latency_estimator;
   std::optional<tensorflow::profiler::ProfiledInstructionsProto> profile =
       ReadPGLEProfile(module, fingerprint);
-  if (profile.has_value()) {
+
+  const bool enable_analytical_latency_estimator =
+      module->config()
+          .debug_options()
+          .xla_gpu_enable_analytical_latency_estimator();
+  if (enable_analytical_latency_estimator) {
```

Comment:
I feel if PGLE is provided, it need to take higher precedence over any in-build latency estimators.

Reply:
that makes sense, changed it to reflect the comment.